src/index.js

```diff
@@ -178,6 +183,23 @@ client.on("messageReactionAdd", async (messageReaction, user) => {
 });
 /* --------------------------------------------------- */
 
+function isDm(message) {
+	return message.channel.type === 'dm';
+}
+
+function handleDmReactionAdd(emoji, message) {
+	if (emoji.name === '❌' && isMessageFromBot(message)) {
+		message.delete().catch(function (error) {
+			console.log(error);
+		});
+		return;
+	}
+}
+
+function isMessageFromBot(message) {
+	return message.author.id === process.env.CLIENT_ID;
+}
+
```

Comment:
Do we need this to be a separate function?
There's also a specific property to check this which evaluates to a boolean:
```js 
message.author.bot
```

Reply:
Not necessary at all, just sugar to make it clearer since `process.env.CLIENT_ID` might not be obvious that it's the bot id at first - moving to the `message.authot.bot` would completely remove that concern though, so I'll use that instead!